### PR TITLE
automattic-texts 0.83.8 (new cask)

### PR DESCRIPTION
**Pull request by PythonCoderAS**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=pullrequests).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/t/texts-com.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+         intel: "2a6748a29f31b143d2e79688d79ccec0b07f52bb180a479a2b28dc62e90e75e4"
+
+  url "https://texts-binaries.texts.com/builds/Texts-macOS-#{arch}-v#{version}-#{hash}.zip"
+  name "Texts.com"
+  desc "DM Manager"
+  homepage "https://texts.com/"
+
+  livecheck do
+    url "https://texts.com/api/install/macos/#{arch}/latest.zip"
+    regex(/Texts-macOS-(?:arm64|x64)-v(\d+(?:\.\d+)*)-[0-9a-f]{10}\.zip$/i)
```
Comment (bevanjkay) [inline Casks/t/texts-com.rb:16]:
```suggestion
    regex(/Texts-macOS-#{arch}[._-]v?(\d+(?:\.\d+)+)-[0-9a-f]{10}\.zip/i)
```

**Review by bevanjkay — COMMENTED** [review]
Thanks @PythonCoderAS 
I have left a small change for the `livecheck` regex.

I can see that there is a token conflict so we cannot go with `texts` as would be the usual token. Does the app show "Texts.com" in the menu bar? Otherwise it may be better to use `automattic-texts`, but happy to discuss and hear other maintainers input also.

**Comment by PythonCoderAS** [comment]
> I can see that there is a token conflict so we cannot go with `texts` as would be the usual token. Does the app show "Texts.com" in the menu bar? Otherwise it may be better to use `automattic-texts`, but happy to discuss and hear other maintainers input also.

It does not. It only says Texts.

**Comment by krehel** [comment]
> I can see that there is a token conflict so we cannot go with `texts` as would be the usual token. Does the app show "Texts.com" in the menu bar? Otherwise it may be better to use `automattic-texts`, but happy to discuss and hear other maintainers input also.

Yeah, I think I agree that `automattic-texts` is more appropriate.

**Review by razvanazamfirei — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**